src/workerd/api/basics.c++

```diff
@@ -495,6 +496,44 @@ bool EventTarget::dispatchEvent(jsg::Lock& js, jsg::Ref<Event> event) {
   return dispatchEventImpl(js, kj::mv(event));
 }
 
+namespace {
+// The jsrpc handler that receives aborts from the remote and triggers them locally
+class AbortSignalImpl final: public rpc::AbortSignal::Server {
```

Comment:
A name like `AbortSignalRpcImpl` would make it clearer how this class differs from the `AbortSignal` API type.